scopesim/utils.py

```diff
@@ -1008,3 +1008,14 @@ def return_latest_github_actions_jobs_status(owner_name="AstarVienna", repo_name
         params_list.append(params)
 
     return params_list
+
+
+def save_unit_to(unit_from: u.Unit, unit_to: u.Unit):
+    """Like unit_from.to(unit_to), but u.um to u.nm == 1000, not 999.999.."""
```

Comment:
Can you state the purpose of the function more explicitely, please? "Like this or that" is a bit vague. Also, the docstring suggests to me that the function should be called `safe_unit_to`, or what is it that is being saved?

Reply:
I'm not very fond of that function `sav/fe_unit_to` - I feel it shouldn't be necessary (why does `Unit.to` do that anyway?). But I did a similar thing when I put in `utils.seq` as an improvement over `numpy.arange`, so I shouldn't say anything other than "use cautiously".